test/validation/api/random/random.c

```diff
@@ -149,6 +149,26 @@ static void random_test_align_and_overflow_true(void)
 	random_test_align_and_overflow(ODP_RANDOM_TRUE);
 }
 
+static void random_test_align_and_len_test(void)
```

Comment:
This looks good. I just wonder if it would be worthwhile to add checks that the function is not writing more than len bytes in the buffer.

Reply:
That and the destination buffer alignment are already tested in random_test_align_and_overflow. So if random test data is allowed to differ for different len values, even if seed is the same, then there's no need for this new test. Removed in v4.